common/scripted_triggers/wc_scripted_triggers.txt

```diff
@@ -640,6 +640,10 @@ can_gain_lose_weight_trigger = {
 # Can't have drunkar trait and drink alcohol
 # Associated with text_effect_can_not_be_drunkard
 can_be_drunkard_trigger = {
```

Comment:
You should look for `can_be_drunkard_trigger` and replace it with `can_drink_alcohol_trigger` where it's needed like in `ZE.8100`: brewmasters can't drink on council meetings right now.

Reply:
i did it in situations where it made sense but not in ZE.8100 since brewmasters are often portrayed as drunk half the time so they probably wouldnt lose control over their movement no matter how drunk